src/LiveComponent/src/Twig/LiveComponentRuntime.php

```diff
@@ -14,7 +14,12 @@
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\UX\LiveComponent\LiveComponentHydrator;
 use Symfony\UX\LiveComponent\Metadata\LiveComponentMetadataFactory;
+use Symfony\UX\StimulusBundle\Helper\StimulusHelper;
 use Symfony\UX\TwigComponent\ComponentFactory;
+use function array_pop;
+use function array_unshift;
+use function explode;
+use function implode;
```

Comment:
This is not in the code style of symfony, afaik ... could you remove them ? :)

Reply:
Ah, default settings in my IDE that does this :)